dd-sdk-android/src/main/kotlin/com/datadog/android/DatadogInterceptor.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android
+
+import com.datadog.android.core.internal.net.CombinedInterceptor
+import com.datadog.android.rum.RumInterceptor
+import com.datadog.android.rum.internal.net.RumRequestInterceptor
+import com.datadog.android.tracing.TracingInterceptor
+import com.datadog.android.tracing.internal.net.TracingRequestInterceptor
+import okhttp3.Interceptor
+import okhttp3.OkHttpClient
+
+/**
+ * Provides automatic integration for [OkHttpClient] by way of the [Interceptor] system.
```

Comment:
```suggestion
 * Provides automatic integration for [OkHttpClient] by using an [Interceptor] implementation.
```

Reply:
Saying it's an Interceptor implementation doesn't give any information, the signature of the class tells that. I actually reused the same javadoc template as other interceptors existing in the wild.

Reply:
Sounds a bit weird for me but if it's already there like that I am fine with it.